File: Assets/Scripts/flySpots.js
#pragma strict

var flySpots : Transform[];
var NumSpots : float = 3;
var randSpot : float;
	

function newSpot (fly : GameObject) {

	
	randSpot = Random.Range(0, NumSpots - 1);
	
	fly.SendMessage("flyHere", flySpots[randSpot]);

}

File: Assets/Scripts/FlySpots.cs
using UnityEngine;
using System.Collections;

public class FlySpots : MonoBehaviour {

    public Vector3[] flySpots;

    public Vector3 NewSpot() 
    {	
	    int randSpot = Random.Range(0, flySpots.Length - 1);
	
	    return flySpots[randSpot];
    }

	void Start () {

	}
	
	void Update () {
	
	}
}
